lib/web/websocket/websocket.js

```diff
@@ -549,6 +549,14 @@ webidl.converters['DOMString or sequence<DOMString>'] = function (V, prefix, arg
   return webidl.converters.DOMString(V, prefix, argument)
 }
 
+webidl.converters.WebSocketInitNodeOptions = webidl.dictionaryConverter([
```

Comment:
I purposefully left this undocumented because it's going to be removed. We don't want to enable permessage-deflate by default until all of the tests pass IMO.